.env_template

```diff
@@ -0,0 +1,9 @@
+# App settings
+CLIENT_HOST=localhost
```

Comment:
Good catch, although `FLASK_BASE_PATH` was replaced by `FLASK_BASE_URL` at some point down the line. I turns out I forgot to change it in the constants file though.